editions/tw5.com/tiddlers/wikitext/HTML in WikiText.tid

```diff
@@ -14,7 +16,36 @@ This is my nice and simple block of text. HelloThere
 </article>
 ```
 
-[[Widgets share the same syntax as HTML tags|Widgets in WikiText]], and so the following information applies to them, too.
+!! Comments
+
+A html-comment as described above has been possible since ~TiddlyWiki Version 5.1.0.
+
+<<.from-version 5.2.0>> It will be possible to use "performant" ''html-comments together with pragmas''. Comments as described below can be cached by the TW core, so they are performant.
+
+```
+<!-- NEW: Comment that describes the macro -->
+\define test()
+some text <!-- inline comment -->
+\end
+
+<!-- This has always been possible -->
```

Comment:
Again I think it confuses matters to try to explain history in this way. All we need is the note above.